src/httpd_jsonapi.c

```diff
@@ -3185,16 +3185,34 @@ jsonapi_reply_library_playlists(struct httpd_request *hreq)
 static int
 jsonapi_reply_library_playlist(struct httpd_request *hreq)
 {
-  const char *playlist_id;
+  uint32_t playlist_id;
   json_object *reply;
   int ret = 0;
 
   if (!is_modified(hreq->req, DB_ADMIN_DB_UPDATE))
     return HTTP_NOTMODIFIED;
 
-  playlist_id = hreq->uri_parsed->path_parts[3];
+  ret = safe_atou32(hreq->uri_parsed->path_parts[3], &playlist_id);
+  if (ret < 0)
+    {
+      DPRINTF(E_LOG, L_WEB, "Could not parse playlist id to integer\n");
+      goto error;
```

Comment:
FYI - having a look at the web changes for playlists for the RSS podcast work, and noticed this - at this point, `reply` is undefined and the `goto` will call `free(reply)`

Reply:
Thanks! Should be fixed now.